isomorphic-fetch/index.d.ts

```diff
@@ -3,182 +3,107 @@
 // Definitions by: Todd Lucas <https://github.com/toddlucas>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-type RequestType = "" | "audio" | "font" | "image" | "script" | "style" |
-    "track" | "video";
-type RequestDestination = "" | "document" | "embed" | "font" | "image" |
-    "manifest" | "media" | "object" | "report" | "script" | "serviceworker" |
-    "sharedworker" | "style" | "worker" | "xslt";
-type RequestMode = "navigate" | "same-origin" | "no-cors" | "cors";
-type RequestCredentials = "omit" | "same-origin" | "include";
-type RequestCache =
-    "default" | "no-store" | "reload" | "no-cache" | "force-cache" |
-    "only-if-cached";
-type RequestRedirect = "follow" | "error" | "manual";
-
-type ResponseType = "basic" | "cors" | "default" | "error" | "opaque" |
-    "opaqueredirect";
-
-type ReferrerPolicy = "" | "no-referrer" | "no-referrer-when-downgrade" |
```

Comment:
According to [spec](https://w3c.github.io/webappsec-referrer-policy/#referrer-policy), a string union is correct. Our lib just has `string`, but maybe we should change the lib instead.